platform/website/src/routes/+layout.svelte

```diff
@@ -32,6 +33,24 @@
 	});
 </script>
 
+<svelte:head>
```

Comment:
On different layout pages, does this get extended?

Reply:
Yes, it does